File: data/python/C_PROGRAM_FACTORIAL_NUMBER_1_MAIN.py
if __name__ == '__main__':
    param = [
    (15,),
    (7,),
    (16,),
    (67,),
    (71,),
    (16,),
    (77,),
    (27,),
    (37,),
    (73,)
        ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("C_PROGRAM_FACTORIAL_NUMBER_1," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("C_PROGRAM_FACTORIAL_NUMBER_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/C_PROGRAM_FACTORIAL_NUMBER_1_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(15);
    param0.add(7);
    param0.add(16);
    param0.add(67);
    param0.add(71);
    param0.add(16);
    param0.add(77);
    param0.add(27);
    param0.add(37);
    param0.add(73);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("C_PROGRAM_FACTORIAL_NUMBER_1," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("C_PROGRAM_FACTORIAL_NUMBER_1," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
